draft-ietf-webtrans-http3.md

```diff
@@ -145,8 +139,8 @@ of !QUIC-DATAGRAM=RFC9221}}).
 {{!RFC8441}} defines an extended CONNECT method in Section 4, enabled by the
 SETTINGS_ENABLE_CONNECT_PROTOCOL setting.  That setting is defined for HTTP/3
 by {{!RFC9220}}.  An endpoint supporting WebTransport over HTTP/3 MUST send
-both the SETTINGS_ENABLE_WEBTRANSPORT setting and the
-SETTINGS_ENABLE_CONNECT_PROTOCOL setting with values set to "1".
+both the SETTINGS_WEBTRANSPORT_MAX_SESSIONS setting with a value greater
+than "0" and the SETTINGS_ENABLE_CONNECT_PROTOCOL setting with a value of "1".
```

Comment:
`SETTINGS_ENABLE_CONNECT_PROTOCOL` is only sent by servers

Reply:
Good call, updated and added a note about the client's value not mattering much in https://github.com/ietf-wg-webtrans/draft-ietf-webtrans-http3/pull/129/commits/0deb77283cc5599741e7dc74e218a7635deecb3a